sources/packages/web/src/services/http/StudentAssessmentApi.ts

```diff
@@ -1,8 +1,8 @@
 import HttpBaseClient from "@/services/http/common/HttpBaseClient";
 import {
-  AssessmentHistorySummaryDTO,
-  RequestAssessmentSummaryApiOutDTO,
-} from "@/types";
+  AssessmentHistorySummaryAPIOutDTO,
+  RequestAssessmentSummaryAPIOutDTO,
+} from "./dto/Assessment.dto";
```

Comment:
Why import is not using @ here?

Reply:
Missed during the refactoring. It is updated now.